pom.xml

```diff
@@ -31,9 +32,11 @@
   </scm>
 
   <properties>
-    <revision>1.14</revision>
+    <revision>1.15</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <revision>1.15</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.60.3</jenkins.version>
+    <jenkins.version>2.164.2</jenkins.version>
```

Comment:
Any particular reason to bump to the latest LTS? If not, I'd prefer to update to 2.138.4 at most since I think that is the oldest LTS line still getting security updates.

Reply:
Not really, the latest LTS just felt like a safe bet. I changed it to 2.138.4.